### DISCUSSION TITLE
add settings to enable/disable batch actions

### DISCUSSION THREAD

**Discussion by paulboudewijn** [directus/directus#9853] [Feature Requests]
Although I like the batch editing options on collections, I'm a little worried certain users will _accidentally_ edit or delete multiple items.

How about enabling batch actions via a setting per collection or user?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
